examples/model_compress/amc/amc_search.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import argparse
+import time
+
+import torch
+import torch.nn as nn
+
+from nni.compression.torch import AMCPruner
+from data import get_split_dataset
+from utils import AverageMeter, accuracy
+
+device = None
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='AMC search script')
+    parser.add_argument('--model_type', default='mobilenet', type=str, help='model to prune')
+    parser.add_argument('--dataset', default='cifar10', type=str, help='dataset to use (cifar/imagenet)')
+    parser.add_argument('--data_root', default='./cifar10', type=str, help='dataset path')
+    parser.add_argument('--sparsity', default=0.5, type=float, help='sparsity of the model')
+    parser.add_argument('--lbound', default=0., type=float, help='minimum sparsity')
+    parser.add_argument('--rbound', default=0.8, type=float, help='maximum sparsity')
+    parser.add_argument('--ckpt_path', default=None, type=str, help='manual path of checkpoint')
+
+    parser.add_argument('--train_episode', default=800, type=int, help='number of training episode')
+    parser.add_argument('--n_gpu', default=1, type=int, help='number of gpu to use')
+    parser.add_argument('--n_worker', default=16, type=int, help='number of data loader worker')
+    parser.add_argument('--data_bsize', default=50, type=int, help='number of data batch size')
```

Comment:
why not use '--batch_size'?

Reply:
Thanks, this is a mistake, will change it to batch_size